pkgs/development/python-modules/qmake2cmake/readme.md

```diff
@@ -0,0 +1,21 @@
+# qmake2cmake
+
+## usage
+
+see examples in
+
+* pkgs/development/python-modules/pyqt/6.x.nix
+* pkgs/development/python-modules/pyqtwebengine/6.x.nix
+
+## debugging
+
+```nix
+stdenv.mkDerivation {
+  cmakeFlags = [ "--debug-output" ];
+  makeFlags = [ "-d" ];
+  # debug qmake2cmake
+  QMAKE2CMAKE_DEBUG_DUMP_FILES = "1"; # *.pro CMakeLists.txt (TODO: cmake_install.cmake)
```

Comment:
should we comment this out before merging?

Reply:
thats just the readme

Reply:
I don't understand this or the comments below. What is it supposed to tell me?

Reply:
with these envs, generated files are dumped to stdout

this is useful when i dont have root access to the builder, and cannot inspect files in the build sandbox